### DISCUSSION TITLE
alternatives to Ali express?

### DISCUSSION THREAD

**Discussion by moonbeast32** [sanni/cartreader#858] [Need help]
Is it just me, or is ali express the worst?

I was able to buy most things on the parts list, but for some things, no matter what I do, my order keeps getting cancelled. I've tried making different accounts, looking for different sellers, using a different browser, etc. The only reason I'm given for these cancellations is "security reasons."

All I have left to get are the MKS LCD, the PIC chipsrt, and all cart slots besides GB and NES. Can these be reliably purchased somewhere else?

**Comment by breyell** [comment]
Howdy! I run Bonzo's Retro Shop. Happy to help you with whatever remaining parts you need, including 3d prints. You can send me a Discord message at bonzo92 or email at [bonzo@bonzosretro.shop](mailto:bonzo@bonzosretro.shop).

**Reply by Ancyker** [reply]
wew, you so fast at replying XD

**Reply by breyell** [reply]
Haha, I just happened to catch this one and saw it's in my wheelhouse.

**Reply by moonbeast32** [reply]
Sure. I will as soon as I get the chance.

**Comment by Ancyker** [comment]
Aliexpress is pretty bad, someone got a built OSCR from an ebay seller that I'm pretty sure is the same one listing on Aliexpress, and uh, well, [that went about as well as you'd expect](https://github.com/sanni/cartreader/discussions/835#discussioncomment-6770793)...
